app/frontend/src/components/Answer/Answer.tsx

```diff
@@ -58,7 +60,9 @@ export const Answer = ({
             </Stack.Item>
 
             <Stack.Item grow>
-                <div className={styles.answerText} dangerouslySetInnerHTML={{ __html: sanitizedAnswerHtml }}></div>
+                <div className={styles.answerText}>
+                    <ReactMarkdown children={sanitizedAnswerHtml} rehypePlugins={[rehypeRaw]} />
```

Comment:
@praveen-palanisamy I'm curious why you brought in the rehypePlugins? I notice a similar repo brings in two other plugins instead, remarkGfm and supersub:
https://github.com/microsoft/sample-app-aoai-chatGPT/blob/354b62b3485ce8270798dd625c4ce904bd311695/frontend/src/components/Answer/Answer.tsx#L63

I'm trying to determine what plugins we need for this repo. Thanks!

Reply:
Thanks for looking into this PR. Yes, preserving the citations and rendering them appropriately is one of the main reasons for choosing this `rehype` plugin.